### PR TITLE
German translation for Lightning Update Cura 4.12

### PR DESCRIPTION
**Pull request by Sekisback**
German translation for the new Ligthning Infill Parameters.
As the Parameters themself are not translated in German right now I'm using my own translation.

switching from own german_translation branch to master, to avoid conflicts  

Finished translation, german translation for the Settings Guide is now completed

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Great to hear! I'm reviewing them, but that's also a lot of work. I'll get it done before the next release.

File: resources/translations/de_DE/troubleshooting/layer_splitting.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Dies ist vor allem ein Problem für die Wände. Die Außenhaut hat eine so große Oberfläche, die nicht zum Reißen neigt. Zwischen Füllung und Wänden sorgen die Wände für den größten Teil der Schichthaftung, da die Füllung oft sehr schnell gedruckt wird, mit spärlich verteilten Linien und oft nicht vertikal gestapelt.
+
+Um einen ausreichenden Druck zu gewährleisten, muss sichergestellt werden, dass keine [Unterextrusion](underextrusion.md) vorliegt. Es gibt alle Ursachen für Unterextrusion, aber einige davon sind für das Layer Splitting von besonderer Bedeutung:
+* Wenn man den Druck auf halber Strecke unterbricht, dauert es eine Weile, bis die Extrusion wieder in Gang kommt. Das "Skript Pause at Height" hat eine Option, um <!--if cura_version >= 4.7-->eine Schicht zu wiederholen<!--endif--><!--if cura_version < 4.7:redo the last few layers-->. Dadurch wird das Material vorbereitet, bevor es nach oben weitergeht, und eine schwache Schicht mit unzureichendem Düsendruck verhindert.
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/troubleshooting/layer_splitting.md]: Missing a translation here for `redo the last few layers`.
Feel free to restructure the sentence if the word order of German doesn't allow it to split off like this.

File: resources/translations/de_DE/troubleshooting/ringing.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Durch diese Anpassungen dauert der Druckvorgang länger. Sie können das Ringing aber auch auf andere Weise verhindern, indem Sie die Art der Bewegungen, die es verursachen, unterbinden:
+* Deaktivieren Sie die Methode [Lücken füllen](../shell/fill_perimeter_gaps.md) und drucken Sie keine [dünnen Wände](../shell/fill_outline_gaps.md), da diese Methoden manchmal sehr kleine Liniensegmente und zusätzliche Fahrbewegungen verwenden. Wenn anschließend eine Wand gedruckt wird, vibriert der Drucker noch ein wenig.
+* Wenn Ihr Drucker nicht den Druckkopf, sondern die Bauplatte auf und ab bewegt, sollten Sie keinen [Z-Sprung] verwenden (../travel/retraction_hop_enabled.md). Diese bewirken, dass die Bauplatte noch lange nach Abschluss des Sprungs auf- und abschwingt.
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/troubleshooting/ringing.md]: This link got broken.
```suggestion
* Wenn Ihr Drucker nicht den Druckkopf, sondern die Bauplatte auf und ab bewegt, sollten Sie keinen [Z-Sprung](../travel/retraction_hop_enabled.md) verwenden. Diese bewirken, dass die Bauplatte noch lange nach Abschluss des Sprungs auf- und abschwingt.
```

**Review by Ghostkeeper — CHANGES_REQUESTED** [review]
I've been skimming all of the texts. These are some possible mistakes I found. I don't know German that well, aside from high school and a bit of personal interest in languages in general. So I've mainly been looking for whether nothing was forgotten, Markdown syntax, copy-paste errors and local inconsistencies.

These are the things I've found. I hope you don't take this as critique. It's natural to make some small mistakes with a huge amount of work like you did!

It's my intention to release these translations with the next update of the Settings Guide, hopefully in about 1.5 weeks. If you don't have time to fix these before then, I might fix some of these up myself as I release it. Most of these have easy suggestions on how we could fix them.

File: resources/translations/de_DE/experimental/adaptive_layer_height_variation.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+Bei einer normalen Schichthöhe von 0,15 mm und einer Variation von 0,1 mm kann Adaptive Layers beispielsweise Schichten mit einer Dicke zwischen 0,05 mm und 0,25 mm erzeugen.
+
+Diese Einstellung schränkt die Leistungsfähigkeit der Funktion "Adaptive Schichten" ein. Wenn der Bereich der Schichtdicken zu eng ist, wird die Schichtdicke immer sehr nahe an der ursprünglichen Schichtdicke liegen, die durch die Einstellung der Schichthöhe festgelegt wurde. Die Funktion "Adaptive Schichten" trägt dann nicht viel zur Zeitersparnis oder Qualitätsverbesserung bei.
+
+Ist der Bereich jedoch sehr groß, können die Schichten sehr dick oder sehr dünn werden. Sehr dicke Schichten erfordern einen starken Ausfluss aus der Düse, was aufgrund der begrenzten Düsengröße und Schmelzkapazität der Heizbahn im Druckkopf nicht immer möglich ist. Sehr dünne Schichten sind aufgrund der Viskosität des Materials oder der Ungenauigkeit der Z-Achse ebenfalls nicht immer möglich. Daher ist es sinnvoll, den Bereich, in dem Adaptive Layers die Schichtdicke einstellen kann, zu begrenzen. Andernfalls wird der Drucker Schwierigkeiten haben, die gewünschte Schichtdicke zu erreichen.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/adaptive_layer_height_variation.md:9]: according to the wording in cura, i changed all "Adaptive Schichten" to "Anpassschichten"

File: resources/translations/de_DE/experimental/bridge_skin_speed_3.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+Wird langsamer gedruckt, ändert sich auch die Geschwindigkeit, mit der das Material aus der Düsenöffnung extrudiert werden muss, erheblich. Diese Änderung der Durchflussrate braucht einige Zeit, was zu einer Überextrusion führt, wenn der Druckkopf für die Brücke langsamer wird, und zu einer Unterextrusion danach. Da die dritte Schicht für die Qualität des Überhangs weniger kritisch ist als die ersten beiden Überbrückungsschichten, ist es besser, die dritte Schicht mit einer Geschwindigkeit zu drucken, die näher an der normalen [Druckgeschwindigkeit] liegt (../speed/speed_topbottom.md), um Über- und Unterextrusion zu vermeiden.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/bridge_skin_speed_3.md:7]: done

File: resources/translations/de_DE/experimental/roofing_angles.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Linienrichtungen der Oberfläche Außenhaut
+====
+Mit dieser Einstellung können Sie für die obersten Schichten der Außenhaut, die durch die Einstellung ["Oberfläche Außenhaut Schichten"](../top_bottom/roofing_layer_count.md) angegeben sind, andere Richtungen für das Muster wählen als für den Rest der Ober- und Unterseite.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/roofing_angles.md:3]: Fixed

File: resources/translations/de_DE/experimental/roofing_line_width.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Oberfläche Außenhaut Linienbreite
+====
+Mit der Einstellung ["Oberfläche Außenhaut Schichten"](../top_bottom/roofing_layer_count.md) können einige Schichten ganz oben eine andere Strichstärke erhalten als die tieferen Schichten der Außenhaut.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/roofing_line_width.md:3]: Fixed

File: resources/translations/de_DE/experimental/roofing_pattern.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+Oberfläche Außenhaut Muster
+====
+Mit dieser Einstellung können Sie für die obersten Schichten der Außenhaut, die durch die Einstellung ["Oberfläche Außenhaut Schichten"](../top_bottom/roofing_layer_count.md) angegeben sind, ein anderes Muster wählen als für den Rest der Ober- und Unterseite.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/roofing_pattern.md:3]: Fixed

File: resources/translations/de_DE/shell/wall_min_flow_retract.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+Einziehen bevorzugt
+====
+Compensating for overlaps in wall lines will reduce the flow rate of some of the walls. If this flow rate gets below the value of the [Durch die Kompensation von Überschneidungen in Wandlinien wird die Durchflussrate einiger Wände reduziert. Wenn diese Flussrate unter den Wert der Einstellung [Mindestwandfluss](wall_min_flow.md) fällt, wird ein Fahrweg anstelle der Wandlinie ausgeführt.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/shell/wall_min_flow_retract.md]: fixed

File: resources/translations/de_DE/shell/z_seam_corner.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+* **Naht offenlegen:** Dadurch wird die Naht bevorzugt an einer Außenecke freigelegt. Wenn die Justierung der Z-Naht auf "Schärfste Kante" eingestellt ist, wird immer die schärfste Außenecke gewählt. Wenn sie auf "Kürzeste" eingestellt ist, wird eine Außenecke gewählt, die sich in der Nähe der aktuellen Position der Düse befindet, wenn sie die Kontur beginnt.
+* **Naht verbergen oder offenlegen:** Dadurch wird eine Naht an einer scharfen Kante erzeugt, unabhängig davon, ob es sich um eine Innen- oder Außenecke handelt, solange sie nicht an einer flachen Wand liegt.
+<!--if cura_version >= 4.2 -->
+* **Intelligent verbergen:** Dies platziert die Naht an einer scharfen Kante, genau wie "Naht verbergen oder offenlegen", aber es werden eher Innenecken als Außenecken bevorzugt, wenn in der Kontur eine Innenecke vorhanden ist. Wenn es keine Innenecken gibt, wird eine Außenecke gewählt..<!--endif-->
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/shell/z_seam_corner.md]: fixed

File: resources/translations/de_DE/shell/z_seam_relative.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
+Realitvwert der Z-Naht
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/shell/z_seam_relative.md]: fixed

File: resources/translations/de_DE/troubleshooting/ringing.md (hunk 2 above)
Comment (Sekisback) [reply resources/translations/de_DE/troubleshooting/ringing.md]: fixed

File: resources/translations/de_DE/troubleshooting/layer_splitting.md (hunk 1 above)
Comment (Sekisback) [reply resources/translations/de_DE/troubleshooting/layer_splitting.md]: fixed

**Comment by Sekisback** [comment]
Hey, all issues fixed. let me know if you found more

**Comment by Ghostkeeper** [comment]
A few of my remarks are still left. I think you may need to expand the 12 hidden conversations.

All of these are pretty easy to solve. Except the one in the Shrinkage Compensation setting article.

File: resources/translations/de_DE/experimental/wipe_hop_enable.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
+Z-Sprung beim Abwischen
+====
+Mit dieser Einstellung wird der Abwischvorgang während des Z-Sprung durchgeführt. Dies kann separat vom normalen [Z-Sprung](../travel/retraction_hop_enabled.md) konfiguriert werden .
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/wipe_hop_enable.md:3]: fixed

File: resources/translations/de_DE/experimental/wipe_pause.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
+bwischen pausieren
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/wipe_pause.md:1]: fixed

File: resources/translations/de_DE/experimental/wipe_retraction_prime_speed.md [hunk 13]
```diff
@@ -0,0 +1,5 @@
+Vorbereitungszeit für Abwischen beim Einzug
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/wipe_retraction_prime_speed.md:1]: you are right, in that case it should be something like "Abwischen Einzuggeschwindigkeit (Zurückschieben)"

File: resources/translations/de_DE/machine_settings/machine_gcode_flavor.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+
+Ultimaker 2
+----
+Diese G-Code-Variante ist für die Ultimaker 2-Familie gedacht. Die Firmware dieser Familie hatte die merkwürdige Idee, dass der Drucker die materialbezogenen Einstellungen steuern sollte und nicht der Slicer. Dies sind die Ausnahmen:
+* Es wird keine Befehle für die Drucktemperatur im G-Code geben.
+* Es wird keine Befehle für die Temperatur der Bauplatte im G-Code geben.
+* Im G-Code wird es keine Befehle für die Temperatur des Bauvolumens geben.
+* Es wird angenommen, dass ersten Düse die [Drucktemperatur für die erste Schicht](../material/material_print_temperature_layer_0.md) hat, wenn der Druck beginnt.
+* Es wird angenommen, dass die Bauplatte die [Temperatur der ersten Schicht](../material/material_bed_temperature_layer_0.md) hat, wenn der Druck beginnt.
+* Der Parameter `E`" des Befehls `G1` wird in Kubikmillimetern berechnet, dem Volumen des zugeführten Kunststoffs, und nicht der Länge des Filaments, das während dieser Bewegung zugeführt wird.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/machine_settings/machine_gcode_flavor.md:35]: done

File: resources/translations/de_DE/machine_settings/machine_max_acceleration_z.md [hunk 15]
```diff
@@ -0,0 +1,7 @@
+Maximum Acceleration Z
+====
+Diese Einstellung gibt die maximale Beschleunigung an, die der Motor und der Rahmen Ihres Druckers in Z-Richtung gemäß der Firmware des Druckers bewältigen können.
+
+Diese Einstellung wird nur verwendet, um korrekte Zeitschätzungen zu erhalten. Die von Cura eingestellte [Druckbeschleunigung](../speed/acceleration_print.md) hat kein Limit, aber Cura geht davon aus, dass Ihre Firmware ein bestimmtes Limit pro Achse setzt. Wenn die Beschleunigung in der ZX-Achse diese Einstellung überschreitet, wird die Zeitschätzung dieser Bewegungen angepasst, um anzunehmen, dass Ihre Firmware die Beschleunigung begrenzt. Die Gesamtbeschleunigung kann immer noch größer sein, wenn mit mehreren Achsen gleichzeitig diagonal beschleunigt wird.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/machine_settings/machine_max_acceleration_z.md:5]: done

File: resources/translations/de_DE/material/infill_material_flow.md [hunk 16]
```diff
@@ -0,0 +1,7 @@
+Fluss der Füllung
+====
+Mit dieser Einstellung wird die Durchflussrate nur für die Füllung angepasst. Die Durchflussrate für die Füllung kann getrennt von der Durchflussrate für den Rest des Drucks eingestellt werden.
+
+Die Anpassung der Durchflussrate der Füllung ist eine Notlösung, um Probleme mit der Extrusionsrate oder der Stärke zu beheben. Der gleiche Effekt kann durch die Anpassung der [Linienabstand Füllung](../infill/infill_line_distance.md) und der [[Linienbreite]](../resolution/infill_line_width.md) der Füllung erzielt werden, diese Einstellung ist jedoch möglicherweise intuitiver.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/material/infill_material_flow.md:5]: done

File: resources/translations/de_DE/material/material_end_of_filament_purge_length.md [hunk 17]
```diff
@@ -0,0 +1,7 @@
+Ausspüldauer am Ende des Filaments
+====
+Einige Drucker sind in der Lage, automatisch eine neue Spule einzulegen, wenn ihnen das Material ausgeht. In diesem Fall müssen sie das Material erneut vorbereiten, bevor sie den Druckvorgang fortsetzen. Mit dieser Einstellung wird die Länge des Filaments festgelegt, das zum Vorbereiten der neuen Spule verwendet wird.
+
+Dies gilt nur, wenn das neue Material dasselbe ist wie das vorherige. Wenn Sie zu einem anderen Material wechseln, wird eine andere[ Ausspüldauer](material_flush_purge_length.md) verwendet.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/material/material_end_of_filament_purge_length.md:5]: done

File: resources/translations/de_DE/material/material_end_of_filament_purge_speed.md [hunk 18]
```diff
@@ -0,0 +1,7 @@
+Ausspülgeschwindigkeit am Ende des Filaments
+====
+Einige Drucker sind in der Lage, automatisch eine neue Spule einzulegen, wenn ihnen das Material ausgeht. In diesem Fall müssen sie das Material erneut vorbereiten, bevor sie den Druckvorgang fortsetzen. Diese Einstellung legt die Geschwindigkeit fest, mit der das Material dann vorbereitet wird. 
+
+Dies gilt nur, wenn das neue Material dasselbe ist wie das vorherige. Beim Wechsel zu einem anderen Material wird eine andere [ Ausspülgeschwindigkeit](material_flush_purge_speed.md) verwendet.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/material/material_end_of_filament_purge_speed.md:5]: done

File: resources/translations/de_DE/material/material_shrinkage_percentage.md [hunk 19]
```diff
@@ -0,0 +1,17 @@
+Kompensation der Schrumpfung des Skalierungsfaktors
+====
+<!--if cura_version >= 4.8-->TMit dieser Einstellung wird das Modell vor dem Slicen automatisch skaliert. Ziel ist es, die Schrumpfung auszugleichen, die beim Abkühlen des Drucks auf Raumtemperatur auftritt. Indem man den Druck etwas größer als gewünscht macht, kann das Endergebnis den ursprünglichen Abmessungen des Eingabemodells besser entsprechen. Dieser Skalierungsfaktor wird auf alle Dimensionen (X, Y und Z) gleichermaßen angewendet.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/material/material_shrinkage_percentage.md:3]: done

File: resources/translations/de_DE/material/material_shrinkage_percentage.md [hunk 20]
```diff
@@ -0,0 +1,17 @@
...
+
+Wenn der Skalierungsfaktor größer als 100,5% ist und Sie etwas Großes drucken, zeigt Cura eine Warnung an, dass es zu Problemen kommen könnte [warping](../troubleshooting/warping.md).<!--endif-->
+
+<!--if cura_version < 4.8:This is a descriptive setting, letting Cura know how much the material shrinks when it cools down from the printing temperature to room temperature.
+
+This setting is currently not used for slicing. It is currently only used to display a warning to the user when printing large things if the shrinkage ratio is larger than 0.5%.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/material/material_shrinkage_percentage.md:15]: done

File: resources/translations/de_DE/platform_adhesion/extruder_prime_pos_y.md [hunk 21]
```diff
@@ -0,0 +1,7 @@
+Extruder Prime Y Position
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/platform_adhesion/extruder_prime_pos_y.md:1]: done

File: resources/translations/de_DE/experimental/wipe_retraction_prime_speed.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (Ghostkeeper) [reply resources/translations/de_DE/experimental/wipe_retraction_prime_speed.md]: Yes! Thank you. I also saw another PR from you there on Cura. Thanks a bunch for those!

File: resources/translations/de_DE/experimental/support_conical_enabled.md [hunk 22]
```diff
@@ -0,0 +1,48 @@
...
+
+Diese Funktion ist eine etwas einfachere und weniger extreme Version von der <!--if cura_version >= 4.7-->[Baumstützstruktur](../support/support_structure.md)<!--endif--><!--if cura_version < 4.7:[Tree Support](support_tree_enable.md)-->. Die Baumstützstruktur hat viele der gleichen Vorteile wie die konische Unterstützung. Die Baumstützstruktur verwendet jedoch einen völlig anderen Algorithmus und lässt mehr Freiheiten bei der Konstruktion des Supports, während die konische Unterstützung eher dem Algorithmus zur Erzeugung des gewöhnlichen Supports folgt. Das bedeutet, dass die Einstellungen für die normale Unterstützung in der Regel besser funktionieren, wenn sie mit der konischen Unterstützung verwendet werden, aber für die Baumstützstruktur müssen sie erheblich angepasst werden. Die Baumstützstruktur spart in der Regel mehr Material und Druckzeit, während die Zuverlässigkeit des Drucks erhalten bleibt.
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/experimental/support_conical_enabled.md:48]: fixed

**Comment by Ghostkeeper** [comment]
Looks good now! Thank you for the corrections.

**Pull request merged**